src/gui/vector/qgsvectorlayerproperties.cpp

```diff
@@ -844,24 +846,24 @@ void QgsVectorLayerProperties::urlClicked( const QUrl &url )
 void QgsVectorLayerProperties::pbnQueryBuilder_clicked()
 {
   // launch the query builder
-  QgsQueryBuilder *qb = new QgsQueryBuilder( mLayer, this );
+  auto *dialog = QgsGui::subsetStringEditorProviderRegistry()->createDialog( mLayer, this );
```

Comment:
Can you skip the auto here please? It's not clear from the surrounds what the return type is

Reply:
done